machine/v1/stable.controlplanemachineset.openstack.testsuite.yaml

```diff
@@ -421,11 +425,35 @@ tests:
             failureDomains:
               platform: OpenStack
               openstack:
-              - availabilityZone: foo
-                rootVolume:
-                  volumeType: a123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345
-    expectedError: "spec.template.machines_v1beta1_machine_openshift_io.failureDomains.openstack[0].rootVolume.volumeType: Too long: may not be longer than 255"
-  - name: Should reject an OpenStack failure domain with an empty rootVolume provided
+              - rootVolume:
+                  availabilityZone: foo
+    expectedError: "spec.template.machines_v1beta1_machine_openshift_io.failureDomains.openstack[0].rootVolume.volumeType: Required value, <nil>: Invalid value: \"null\": some validation rules were not checked"
+  - name: Should reject an OpenStack failure domain with an empty rootVolume volumeType provided
+    initial: |
+      apiVersion: machine.openshift.io/v1
+      kind: ControlPlaneMachineSet
+      spec:
+        selector:
+          matchLabels:
+            machine.openshift.io/cluster-api-machine-role: master
+            machine.openshift.io/cluster-api-machine-type: master
+        template:
+          machineType: machines_v1beta1_machine_openshift_io
+          machines_v1beta1_machine_openshift_io:
+            metadata:
+              labels:
+                machine.openshift.io/cluster-api-machine-role: master
+                machine.openshift.io/cluster-api-machine-type: master
+                machine.openshift.io/cluster-api-cluster: cluster
+            spec:
+              providerSpec: {}
+            failureDomains:
+              platform: OpenStack
+              openstack:
+              - rootVolume:
+                  volumeType: ""
+    expectedError: "spec.template.machines_v1beta1_machine_openshift_io.failureDomains.openstack[0].rootVolume.volumeType: Invalid value: \"\""
```

Comment:
What follows on from this? I think it's important to capture the message that explains how to fix the error

Reply:
done